Upstream-Status: Backport

Fix for stricter Makefile parser in Make 3.82 take from Gentoo bugzilla:
http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/sys-process/procps/files/procps-3.2.8%2Bgmake-3.82.patch?revision=1.1

Index: procps-3.2.8/Makefile
===================================================================
--- procps-3.2.8.orig/Makefile
+++ procps-3.2.8/Makefile
@@ -174,7 +174,8 @@ INSTALL := $(BINFILES) $(MANFILES)
 # want this rule first, use := on ALL, and ALL not filled in yet
 all: do_all
 
--include */module.mk
+-include proc/module.mk
+-include ps/module.mk
 
 do_all:    $(ALL)
 
